.github/workflows/stale-bot.yaml

```
@@ -0,0 +1,22 @@
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
```

`contents: write` is only for [`delete-branch: true`](https://github.com/actions/stale#delete-branch) (default is `false`):


```suggestion
    permissions:
      issues: write
      pull-requests: write
```